conf/app.routes

```diff
@@ -2,9 +2,9 @@
 
 ->         /hmrc-frontend        hmrcfrontend.Routes
 
-GET        /product-details      uk.gov.hmrc.incometaxsoftwarechoicesfrontend.controllers.ProductDetailsController.show(software: Option[String] ?= None)
-GET        /                     uk.gov.hmrc.incometaxsoftwarechoicesfrontend.controllers.SearchSoftwareController.show
-POST       /                     uk.gov.hmrc.incometaxsoftwarechoicesfrontend.controllers.SearchSoftwareController.search(ajax: Boolean = false)
-POST       /ajax                 uk.gov.hmrc.incometaxsoftwarechoicesfrontend.controllers.SearchSoftwareController.search(ajax: Boolean = true)
-GET        /glossary             uk.gov.hmrc.incometaxsoftwarechoicesfrontend.controllers.GlossaryController.show(ajax: Boolean = true)
-GET        /assets/*file         controllers.Assets.versioned(path = "/public", file: Asset)
+GET        /product-details/*software uk.gov.hmrc.incometaxsoftwarechoicesfrontend.controllers.ProductDetailsController.show(software: String)
```

Comment:
Structural change for URL.  This is needed to make referer analysis work correctly.